lib/excon.rb

```diff
@@ -116,7 +116,7 @@ def new(url, params = {})
         :port       => uri.port,
         :query      => uri.query,
         :scheme     => uri.scheme
-      }.merge!(params)
```

Comment:
Fair. Yeah, we fixed the json issue on master I think, so hopefully that shouldn't be an issue moving forward.